ui/src/shared/components/TableGraph.js

```diff
@@ -0,0 +1,71 @@
+import React, {PropTypes, Component} from 'react'
+import _ from 'lodash'
+import {timeSeriesToTable} from 'src/utils/timeSeriesToDygraph'
+import {MultiGrid} from 'react-virtualized/dist/commonjs/MultiGrid'
+
+class TableGraph extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  componentWillMount() {
+    this._labels = []
+    this._data = [[]]
```

Comment:
Is there a reason for phrasing these as instance variables as opposed to component state? These don't have the benefit of an automatic component rerender if they change